index.html

```diff
@@ -0,0 +1,244 @@
+                javascript
+              </li>
+            </ul>
+          </div>
+          <div>
+            <button> See Project</button>
+          </div>
+        </div>
+
+        <div class="grid column row">
+          <div class="three"></div>
+          <div>Tonic</div>
+          <div>
+            <div>CANOPY</div>
+            <ul class="flex">
+              <li>Back End Dev</li>
+              <li>2015</li>
+            </ul>
+          </div>
+          <div>
+            A daily selection of privately personalized reads;
+            no accounts or sign-ups required.
+          </div>
+          <div>
+            <ul class="flex">
+              <li>
+                html
+              </li>
+              <li>
+                css
+              </li>
+              <li>
+                javascript
+              </li>
+            </ul>
+          </div>
+          <div>
+            <button> See Project</button>
+          </div>
+        </div>
+
+        <div class="grid column row">
+          <div class="four"></div>
+          <div>Multi-Post Stories</div>
+          <div>
+            <div>CANOPY</div>
+            <ul class="flex">
+              <li>Back End Dev</li>
+              <li>2015</li>
+            </ul>
+          </div>
+          <div>
+            A daily selection of privately personalized reads;
+            no accounts or sign-ups required.
+          </div>
+          <div>
+            <ul class="flex">
+              <li>
+                html
+              </li>
+              <li>
+                css
+              </li>
+              <li>
+                javascript
+              </li>
+            </ul>
+          </div>
+          <div>
+            <button> See Project</button>
+          </div>
+        </div>
+
+        <div class="about grid row2 column">
+          <section>
+            <h1>About<br>Myself</h1>
+            <p>
+              Hello I’m a software developer! 
+              I can help you build a product,
+              feature or website Look through 
+              some of my work and experience!
+              If you like what you see and have 
+              a project you need coded, 
+              don’t hestiate to contact me.
+            </p>
+            <div class="connect">LET'S CONNECT</div>
+            <div>
+              <ul class="flex">
+                 <li>
+                   <a href="#"><img src="icons/space.svg" alt="icon"></a>
+                 </li>
+                 <li>
+                   <a href="#"><img src="icons/git.svg" alt="icon"></a>
+                 </li>
+                 <li>
+                   <a href="#"><img src="icons/chi.svg" alt="icon"></a>
+                 </li>
+                 <li>
+                   <a href="#"><img src="icons/twitter.svg" alt="icon"></a>
+                 </li>
+              </ul>
+            </div>
+            <button>
+              Get my resume
+            </button>
```

Comment:
Also, you should implement button effects on hover, active, and pressed for more details check the effects on the comment above